app/models/miq_server/worker_management/monitor.rb

```diff
@@ -99,6 +99,12 @@ def check_not_responding(class_name = nil)
     processed_workers.collect(&:id)
   end
 
+  NOT_RESPONDING = :not_responding
+  MEMORY_EXCEEDED = :memory_exceeded
```

Comment:
I feel like this information belongs somewhere else. Maybe `MiqServer::WorkerManagement::Monitor::Reason`?

Then ideally callers of `worker_set_monitor_reason` will also use the same constants. Not sure if making that kind of change is in this PR's scope though.